click_odoo_contrib/manifest.py

```diff
@@ -69,18 +69,22 @@ def add_deps(name):
     res = set()
     for module_name in module_names:
         add_deps(module_name)
-    if include_active:
-        for module_name in sorted(odoo.modules.module.get_modules()):
-            module_path = odoo.modules.get_module_path(module_name)
-            manifest = read_manifest(module_path)
-            if manifest.get("active"):
-                add_deps(module_name)
     if include_auto_install:
         auto_install_list = []
         for module_name in sorted(odoo.modules.module.get_modules()):
             module_path = odoo.modules.get_module_path(module_name)
             manifest = read_manifest(module_path)
-            if manifest.get("auto_install"):
+            # "auto_install" can be a list or a boolean
+            # see Odoo's documentation for Module Manifests
```

Comment:
This part seems unrelated to the issue, and should be best done in another PR.
BTW, where is this documented?

Reply:
NVM, I found the comment in the Odoo codebase.